poppy/geometry.py

```diff
@@ -35,9 +35,9 @@ def _arc(x, y0, y1, r):
     positive.
     """
     if accel_math._USE_NUMEXPR:
-        return ne.evaluate("0.5 * r**2 * (arctan(y1/x) - arctan(y0/x))")
+        return ne.evaluate("0.5 * r**2 * (arctan2(y1/x) - arctan2(y0/x))")
```

Comment:
The syntax with arctan2 need two parameters separated by a comma: `arctan2(numerator, denominator)`, instead of a single parameter that's a ratio.

Reply:
Thank you, I threw together this PR too quickly. I have made the change.